### ISSUE TITLE
Enhancement: Fixing the footer UI

### ISSUE THREAD

**Issue by agayushh** [AvgBlank/URLShortener#23]
The footer seems like going out of the website 

![image](https://github.com/user-attachments/assets/577df861-95f7-4b93-b251-e016a4b45d41)
![image](https://github.com/user-attachments/assets/f9ee0941-e021-482d-a50d-62a42c1bfd51)

**Comment by agayushh** [comment]
@AvgBlank please assign me on this one

**Comment by AvgBlank** [comment]
Yup!

**Comment by SparshKhanna0001** [comment]
is this resolved...?

**Comment by AvgBlank** [comment]
Nope, not yet, want me to assign you instead?

**Comment by SparshKhanna0001** [comment]
yea, before that it would be really great if you very precisely describe what is the suitable footer that you want...!

**Comment by SparshKhanna0001** [comment]
Do i need to fix the last links so that they don;t go out of the boundary, as they do now...?

**Comment by agayushh** [comment]
yeah i am unassigning myself you can work upon it as it is not setting up on my local system

**Comment by AvgBlank** [comment]
> Do i need to fix the last links so that they don;t go out of the boundary, as they do now...?

Yes, just move it out of way.

**Comment by SparshKhanna0001** [comment]
assign me.

**Comment by SparshKhanna0001** [comment]
i did made the changes tho, the push request is causing some issues...

**Comment by SparshKhanna0001** [comment]
pull request has been made...

**Comment by Sachin-Ram** [comment]
Is the issue resolved?
